app.rb

```diff
@@ -0,0 +1,102 @@
+class App
+  def run
+    puts 'Welcome to Catalog of Things!'
+    loop do
+      menu
+      option = get.chomp
```

Comment:
Hello @Lockless404 ,
I know your project has been approved but as a code checker I later examined that the app wasn't running properly, the column below shows the error

![apprb](https://user-images.githubusercontent.com/83566118/159375500-1de33424-c3ad-4ee2-b9c7-26a1bda6badd.PNG)

- [ ] Please go back to your `app.rb` file  **line-6** apply 
```Ruby 
option = gets.chomp.to_i 
```
Cheers and Happy coding!👏👏👏